src/marqo/version.py

```diff
@@ -1,13 +1,8 @@
-__marqo_version__ = "2.9.0"
-__marqo_release_page__ = f"https://github.com/marqo-ai/marqo/releases/tag/{__marqo_version__}"
-
-__minimum_supported_marqo_version__ = "2.6.0"
-
+__minimum_supported_marqo_version__ = "2.9.0"
 
+# NOTE: This isn't used anywhere
```

Comment:
So why do we need this if it's not used in any places?

Reply:
I didn't totally remove `def supported_marqo_version()` yet as someone might have a comment on its use. 

`__marqo_version__` is removed though.